vault/provider.go

```diff
@@ -629,6 +629,18 @@ var (
 			Resource:      quotaRateLimitResource(),
 			PathInventory: []string{"/sys/quotas/rate-limit/{name}"},
 		},
+		"vault_terraform_cloud_secret_backend": {
+			Resource:      terraformCloudSecretBackendResource(),
+			PathInventory: []string{"/terraform/config"},
+		},
+		"vault_terraform_cloud_secret_creds": {
+			Resource:      terraformCloudSecretCredsResource(),
+			PathInventory: []string{"/terraform/creds/{role}"},
+		},
+		"vault_terraform_cloud_secret_role": {
+			Resource:      terraformCloudSecretRoleResource(),
+			PathInventory: []string{"/terraform/roles/{name}"},
```

Comment:
Should it be `]string{"/terraform/role/{name}"}` instead?

Reply:
Fixed.